paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -158,6 +160,9 @@ class CustomerSheet @Inject internal constructor(
          * Your customer-facing business name. The default value is the name of your app.
          */
         val merchantDisplayName: String? = null,
+
+        // TODO(tillh-stripe) Add docs
+        internal val preferredNetworks: List<CardBrand> = emptyList(),
```

Comment:
Might need to add a note here in case `CustomerSheet` releases first in order to keep the CBC functionality hidden.

Reply:
It’s hidden already, as we don’t expose the constructor. I can eventually make `Configuration.preferredNetworks` public for consistency with the other fields, but I think we should keep them internal by default.